lib/Command/ExpirePassword.php

```diff
@@ -106,54 +129,128 @@ protected function configure() {
 	 * @throws \OCP\PreConditionNotMetException
 	 */
 	protected function execute(InputInterface $input, OutputInterface $output) {
-		$uid = $input->getArgument('uid');
+		$groups = $input->getOption('group');
+		$allUsers = $input->getOption('all');
+
+		if ($allUsers !== false) {
+			$users = [];
+			$function = function (IUser $user) use (&$users) {
+				$users[] = $user;
+			};
+
+			$this->userManager->callForAllUsers($function);
+			$this->setPasswordExpiryForUsers($input, $output, $users);
+		} elseif (\count($groups) >= 1) {
+			$users = [];
+
+			foreach ($groups as $group) {
+				if ($this->groupManager->groupExists($group) === false) {
```

Comment:
It's a bit better to check for `true` because we want to process the groups only if the group exists. If the function returns something else, we don't want to process the group.
(Just saying, as I expect some changes around and this one is easy. It isn't important because the method should only return true or false)

Reply:
In the latest version I removed the check to false. And used `true`. Also made sure that if the group doesn't exist, it wouldn't return from the `execute` method. The list of groups will be parsed and processed.